com.microsoft.mrtk.accessibility/Utilities/ObjectClassification.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.Accessibility
+{
+    /// <summary>
+    /// Classifications for describable objects that may appear in the scene.
+    /// </summary>
+    [Flags]
+    public enum ObjectClassification
```

Comment:
This feels like a bit of a weird thing to me, since we don't support more than one classification per-object, but I don't really have a better suggestion 😅

Reply:
we _could_ require the screen reader to query each desired classification individually, though that would change the clearing logic (not an issue, just puts the burden on the reader, not the object registry)